torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -314,3 +314,96 @@ def resized_crop_image_pil(
 ) -> PIL.Image.Image:
     img = crop_image_pil(img, top, left, height, width)
     return resize_image_pil(img, size, interpolation=interpolation)
+
+
+class _FiveCropResult(NamedTuple):
```

Comment:
Why introduce a new private named duple? I think we can keep the simple tuple approach currently used on main:
https://github.com/pytorch/vision/blob/b8de0b84c2382ce237f801e141d954c1deaa090c/torchvision/transforms/functional.py#L751